src/runtime_src/core/common/api/aie/xrt_graph.cpp

```diff
@@ -228,6 +228,76 @@ send_exception_message(const char* msg)
   xrt_core::message::send(xrt_core::message::severity_level::error, "XRT", msg);
 }
 
+static void
+configure_bd(xrtDeviceHandle dhdl, int tileType, uint8_t column, uint8_t row, uint8_t bdId,
```

Comment:
Regarding all of these wrappers, there must be a better way to do so with possibly templates.
@stsoe Do you have any ideas?

Reply:
ADF API must change.  We cannot possibly support this.
Else, we have to provide a struct at the lowest common denominator and then leave it to clients to populate the struct which is then an argument to the funciton.

I would like to understand why these APIs need to be reflected in XRT graph / aie implementation.  Who uses them and how?